src/containerapp/azext_containerapp/_utils.py

```diff
@@ -532,7 +532,7 @@ def _is_valid_weight(weight):
 
 
 def _update_traffic_weights(containerapp_def, list_weights):
-    if "traffic" not in containerapp_def["properties"]["configuration"]["ingress"] or list_weights and len(list_weights):
+    if "traffic" not in containerapp_def["properties"]["configuration"]["ingress"] or not containerapp_def["properties"]["configuration"]["ingress"]["traffic"] or list_weights and len(list_weights):
```

Comment:
> containerapp_def["properties"]["configuration"]["ingress"]["traffic"]

I want to confirm that you can ensure that there is no case where a parent attribute is `None`?

Reply:
remove this a part of this release

Reply:
removed